packages/carbon-graphs/CHANGELOG.md

```diff
@@ -2,4 +2,7 @@
 
 ## Unreleased
 
+* Changed
+  * Replaced `document` in all draw helpers with `graph.legendSVG` for better control.
```

Comment:
nit: use `-` for consistency

Reply:
also nit:  move to below `Migrated to the terra-graphs mono-rep.` so that this is in chronological order.

Reply:
Updated here https://github.com/cerner/terra-graphs/pull/18/commits/254cf8305dbc7fbc60951bec774e787e3ac8252a.